tools/run-ci.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+set -ev
+
+# always change the working directory to the project's root directory
+cd $(dirname $0)/..
+
+if [ "${LINTER_ONLY}" = "true" ]; then
```

Comment:
you may want `${LINTER_ONLY-}` just in case the `u` option is set

Reply:
I'm not sure what you mean - in case `set -u` is used in the bash script?